docs/pages/abide.md

```diff
@@ -61,7 +61,7 @@ These input types create a text field: `text`, `date`, `datetime`, `datetime-loc
     <div class="grid-x grid-margin-x">
       <div class="cell large-6">
         <label>URL Pattern, not required, but throws error if it doesn't match the Regular Expression for a valid URL.
-          <input type="text" placeholder="https://foundation.zurb.com" pattern="url">
+          <input type="text" placeholder="https://www.get.foundation" pattern="url">
```

Comment:
Do we need the www st the beginning? I would prefer the short version. Easier to type and shorter inn general.